consensus/babe/babe.go

```diff
@@ -201,11 +203,28 @@ func (b *Session) vrfSign(input []byte) ([]byte, error) {
 	return out, err
 }
 
-// BuildBlock Builds the block
-func (b *Session) buildBlock(number *big.Int) (*types.Block, error) {
-	block := types.Block{
-		Header: types.BlockHeaderWithHash{Number: number},
-		Body:   []byte{1, 2, 3, 4, 5},
+// Block Build
+func (b *Session) buildBlock(parent *types.BlockHeaderWithHash, slot Slot) (*types.Block, error) {
+	log.Debug("build-block", "parent", parent, "slot", slot)
+
+	// Initialize block
+	encodedHeader, err := codec.Encode(parent)
+	if err != nil {
+		return nil, err
+	}
+	err = b.initializeBlock(encodedHeader)
+	if err != nil {
+		return nil, err
+	}
+
+	// TODO: inherents and extrinsics
```

Comment:
Is this what needs to be finished for build block?

Reply:
yeah I have issues open for them #500 #501 
I did #500 already in my other PR that I can merge into here if you'd like

Reply:
how you merge is up to you, just wanted to make sure we had an issue open for it

Reply:
okay gonna leave that PR open for now, should have issues open covering what still needs to be done